### PR TITLE
Added --legacy-peer-deps flag to install cli.

### PR DESCRIPTION
**Pull request by kawikabader**
## Summary

The `create` command was failing when using npm7.

## Changelog

Added `--legacy-peer-deps` flag to install cli.
## Test Plan

Manually tested using npm6 and npm7.

### CODE REVIEW

**Review by burgessa23 — APPROVED** [review]

**Review by kinetifex — APPROVED** [review]

File: packages/gasket-utils/lib/package-manager.js [hunk 1]
```diff
@@ -139,7 +139,9 @@ class PackageManager {
    * @public
    */
   async install(args = []) {
-    return this.exec('install', args);
+    // Installing with --legacy-peer-deps flag to accommodate npm7, specifically
+    // requiring different versions of react
+    return this.exec('install', [...args, '--legacy-peer-deps']);
```
Comment (rmarkins-godaddy) [inline packages/gasket-utils/lib/package-manager.js:144]: is this backwards comapatible? aka if i did this w/ `npm@6` will it still add `--legacy-peer-deps`?

Comment (rmarkins-godaddy) [reply packages/gasket-utils/lib/package-manager.js:144]: Should we also add tests around this?

**Comment by rmarkins-godaddy** [comment]
@kinetifex @kawikabader did we try `--force` first prior to `--legacy-peer-deps`. I believe that would be more backward compat if someone was to use `npm@6` to use `gasket/cli`

File: packages/gasket-utils/lib/package-manager.js (hunk 1 above)
Comment (kinetifex) [reply packages/gasket-utils/lib/package-manager.js:144]: `--force` has other implications, whereas `--legacy-peer-deps` handles the specific issue at hand.

Comment (kawikabader) [reply packages/gasket-utils/lib/package-manager.js:144]: > is this backwards comapatible? aka if i did this w/ `npm@6` will it still add `--legacy-peer-deps`?. iirc `npm@6` does not have this method. If you want to go this route you will want to check the `npm` version in use prior to this return

I did test this with npm@6 and had no issues installing.

**Review by rmarkins-godaddy — APPROVED** [review]

**Pull request merged**